src/Scripting/Core/Hosting/CommonMemberFilter.cs

```diff
@@ -13,6 +13,11 @@ internal class CommonMemberFilter : MemberFilter
     {
         public override bool Include(StackFrame frame)
         {
+            if (frame.GetMethod().DeclaringType.FullName.StartsWith("Submission#0"))
```

Comment:
>if (frame.GetMethod().DeclaringType.FullName.StartsWith("Submission#0")) [](start = 12, length = 72)

Could you please file an issue tracking improving this as we discussed and add a TODO comment to the code that links to that issue?